mssqlcli/mssql_cli.py

```diff
@@ -667,6 +687,17 @@ def has_change_db_cmd(self, query):
 
         return False, None
 
+    def has_security_statement_in_cmd(self, query):
+        """Determines if the statement contains a PASSWORD or SECRET keyword
+        """
+        try:
+            tokens = query.lower()
+            keywords_flagged = [keyword for keyword in security_keywords if keyword in tokens]
+            return keywords_flagged
```

Comment:
Isn't this the same logic defined in the MssqlFileHistory class? It'd be nice to put the logic in one place, as simple as the logic may be.

Reply:
I'll define it as a module level method that both functions can reference.